README.md

```diff
@@ -25,6 +25,59 @@ This is not designed to be realtime. Postoffice uses [GenStage](https://github.c
 * Cluster nodes. Best effort to avoid sending messages more than once. (More info on clustering later)
 * Endpoint to be used as health-check from k8s `/api/health`
 
+## API
+We expose an API to enable projects to create the structure they need to work: topics, publishers and messages.
+For both topics and publishers, if the resource already exist we return `400 bad request`.
```

Comment:
Postoffice returns 400 when the payload is invalid too, for example, when try to create a topic without `name`

Reply:
Yes but that case makes sense as the request is wrong. Giving `bad request`is something not expected in cases where the respurce already exists. There is an open issue #46 to change that behaviour